### PR TITLE
[browser][tests] Standup System.Net.WebSockets.Client.Tests in CI

### PR DESCRIPTION
**Pull request by kjpou1**
Part of https://github.com/dotnet/runtime/issues/44666

Outstanding for now are the tests that use LoopBack which depends on websockets and are not supported on browser.  Marked the loopback issues as activeIssue - `[ActiveIssue("#34690", TestPlatforms.Browser)]`

Also, one marked `[ActiveIssue("https://github.com/dotnet/runtime/issues/44720")]` which is being looked into.  https://github.com/dotnet/runtime/issues/44720

### CODE REVIEW

**Review by lewing — COMMENTED** [review]
Looks ok after the loopback activeissue change

File: src/libraries/System.Net.WebSockets.Client/tests/CloseTest.cs [hunk 1]
```diff
@@ -327,6 +327,7 @@ public async Task CloseAsync_DuringConcurrentReceiveAsync_ExpectedStates(Uri ser
 
         [ConditionalFact(nameof(WebSocketsSupported))]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/34690", TestPlatforms.Windows, TargetFrameworkMonikers.Netcoreapp, TestRuntimes.Mono)]
+        [PlatformSpecific(~TestPlatforms.Browser)] // LoopBack not not supported on browser
```
[the remark this answers is not in the document]
Comment (kjpou1) [reply src/libraries/System.Net.WebSockets.Client/tests/CloseTest.cs]: Addressed in recent commits

File: src/libraries/System.Net.WebSockets.Client/tests/CloseTest.cs [hunk 2]
```diff
@@ -327,6 +327,7 @@ public async Task CloseAsync_DuringConcurrentReceiveAsync_ExpectedStates(Uri ser
 
         [ConditionalFact(nameof(WebSocketsSupported))]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/34690", TestPlatforms.Windows, TargetFrameworkMonikers.Netcoreapp, TestRuntimes.Mono)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/34690", TestPlatforms.Browser)]
```
Comment (lewing) [inline src/libraries/System.Net.WebSockets.Client/tests/CloseTest.cs:330]: These can all be 

```suggestion
        [ActiveIssue("https://github.com/dotnet/runtime/issues/34690", TestPlatforms.Windows | TestPlaforms.Browser, TargetFrameworkMonikers.Netcoreapp, TestRuntimes.Mono)]
        [ActiveIssue("https://github.com/dotnet/runtime/issues/34690", TestPlatforms.Browser)]
```

But I also made an error in my first review comment and meant https://github.com/dotnet/runtime/issues/42852

**Review by stephentoub — APPROVED** [review]
Other than the existing comments, LGTM.

File: src/libraries/System.Net.WebSockets.Client/tests/CloseTest.cs (hunk 2 above)
Comment (kjpou1) [reply src/libraries/System.Net.WebSockets.Client/tests/CloseTest.cs]: Addressed in recent commits

**Comment by kjpou1** [comment]
Thanks for the catch Larry : Make browser specific failure browser specific

**Review by lewing — APPROVED** [review]

**Pull request merged**